qute-web-markdown/runtime/src/main/java/io/quarkiverse/qute/web/markdown/runtime/commonmark/CommonMarkConverter.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkiverse.qute.web.markdown.runtime.commonmark;
+
+import org.commonmark.node.Node;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
+
+import io.quarkiverse.qute.web.markdown.runtime.MarkdownConverter;
+
+public class CommonMarkConverter implements MarkdownConverter {
+
+    private static final Parser PARSER = Parser.builder().build();
```

Comment:
Why `static`? There's only one `CommonMarkConverter` instance shared for all instances so I believe that `private final` is enoug.